source/docs/09-textencoding.xml

```diff
@@ -470,16 +471,421 @@
                      </figure>
                   </p>
                </div>
-               <div xml:id="sharedNames" type="div3">
-                  <head>Names</head>
-                  <p>The <gi scheme="MEI">name</gi> element may be used to mark up portions of a text that function as name.</p>
+            </div>
+            <div xml:id="sharedNames" type="div2">
```

Comment:
I disagree, because we should avoid to many subsections like `9.2.8.3.5.`. The previous state was a result of the whole chapter being cramped in there.

Reply:
Will this be for MEI 6? If not, then we should certainly not change the structure because we have only on version of the guidelines per major release.

Reply:
So you don't want this to be back in the Guidelines?

Reply:
No, what I am saying is that we should not change the div levels.